# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../p3nfs/p3nfs.devfs.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -ruN p3nfs-5.16/configure.in p3nfs-5.16-new/configure.in
--- p3nfs-5.16/configure.in	2003-06-11 00:49:13.000000000 +0200
+++ p3nfs-5.16-new/configure.in	2005-02-20 15:18:52.000000000 +0100
@@ -72,7 +72,7 @@
 
 
 AC_MSG_CHECKING(for default serial line)
-for i in /dev/tty0p0 /dev/tty00 /dev/ttyS0 /dev/term/a /dev/ttyfa /dev/ttya /dev/tty0 /dev/ttyf1 /dev/cuaa1 ; do
-  if test -c $i ; then
+for i in /dev/ttyS0 ; do
+  if test -n "$i" ; then
     DDEV=$i
     AC_MSG_RESULT($i)
